test/WorkflowCore.Tests.SqlServer/SqlServerQueueProviderFixture.cs

```diff
@@ -0,0 +1,140 @@
+﻿#region using
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using WorkflowCore.Interface;
+using WorkflowCore.QueueProviders.SqlServer.Services;
+
+using Xunit;
+using Xunit.Abstractions;
+
+#endregion
+
+namespace WorkflowCore.Tests.SqlServer
+{
+    [Collection("SqlServerBroker collection")]
```

Comment:
If you use the `SqlServer Collection`, you can use the docker image to run tests against, instead of requiring a local instance